src/filterBar/FilterBar.story.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+import { action, boolean, select } from '@storybook/addon-actions'
+import { color } from '../_utils/branding'
+import { CarpoolIcon } from '../icon/carpoolIcon'
+import { BusIcon } from '../icon/busIcon'
+import { FilterBar, FilterBarProps } from '../filterBar'
+
+<Meta title="Widget|FilterBar" />
+
+# FilterBar
+
+export const supplyInfo = [
```

Comment:
Is this array visible in the source code of the story?

Reply:
cc: @gabrielagabriel

Reply:
Yes it's the one from the screenshot

Reply:
Ok 👍